pkg/fetch/vendir.go

```diff
@@ -13,10 +13,10 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/apis/kappctrl/v1alpha1"
+	"carvel.dev/kapp-controller/pkg/apis/kappctrl/v1alpha1"
 	// we run vendir by shelling out to it, but we create the vendir configs with help from a vendored copy of vendir.
+	"carvel.dev/kapp-controller/pkg/exec"
```

Comment:
nit: The comment should be for the vendirconf import

Reply:
fixed